test/definitions/super-agent/logs/amazonlinux-2023-super-agent.logs.json

```diff
@@ -0,0 +1,39 @@
+{
+    "global_tags": {
+      "owning_team": "virtuoso",
+      "Environment": "development",
+      "Department": "product",
+      "Product": "virtuoso"
+    },
+  
+    "resources": [
+      {
+        "id": "host1",
+        "provider": "aws",
+        "type": "ec2",
+        "size": "t3.small",
+        "ami_name": "al2023-ami-2023.*-aarch64",
```

Comment:
I couldn't find an AMI with the specified name. Could you please check if it exists?